vcs.go

```diff
@@ -92,7 +96,8 @@ func (g Git) ReadDir(revision, path string) ([]os.FileInfo, error) {
 	return files, nil
 }
 
-func (g Git) OpenFile(revision, path string) (io.ReadCloser, error) {
+// OpenFile returns a reader for a given absolute path at a revision
+func (g *Git) OpenFile(revision, path string) (io.ReadCloser, error) {
```

Comment:
Doesn't *need* to be, but you're returning a *Git so all your functions should be pointer receivers for consistency